Gyoza.qmd

```diff
@@ -0,0 +1,37 @@
+---
+title: "Gyoza"
+image: "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2Fcrispy-pork-gyoza--395542779777568469%2F&psig=AOvVaw2ipnJCY1OVUTrS-coBjojI&ust=1718103802111000&source=images&cd=vfe&opi=89978449&ved=0CBIQjRxqFwoTCPCWhM_x0IYDFQAAAAAdAAAAABAJ"
+image-alt: "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2Fcrispy-pork-gyoza--395542779777568469%2F&psig=AOvVaw2ipnJCY1OVUTrS-coBjojI&ust=1718103802111000&source=images&cd=vfe&opi=89978449&ved=0CBIQjRxqFwoTCPCWhM_x0IYDFQAAAAAdAAAAABAJ"
+categories:
+  - vegeterian
+  - vegan
+  - Asian
+  - Date Night Food
```

Comment:
@KaroBrix could you please add your name as an author to the YAML header, like this for example:

```YAML
author: Karo Brix
```